src/pgen/torus.cpp

```diff
@@ -197,6 +266,69 @@ void ProblemGenerator(MeshBlock *pmb, ParameterInput *pin) {
             v(irho, k, j, i), v(ieng, k, j, i) / v(irho, k, j, i), lambda);
         v(iprs, k, j, i) = eos.PressureFromDensityTemperature(v(irho, k, j, i),
                                                               v(itmp, k, j, i), lambda);
+        v(igm1, k, j, i) = eos.BulkModulusFromDensityTemperature(
+                               v(irho, k, j, i), v(itmp, k, j, i), lambda) /
+                           v(iprs, k, j, i);
+
+        Real ucon[4] = {0};
+        Real vpcon[3] = {v(ivlo, k, j, i), v(ivlo + 1, k, j, i), v(ivlo + 2, k, j, i)};
+        Real gcov[4][4] = {0};
+        geom.SpacetimeMetric(CellLocation::Cent, k, j, i, gcov);
+        GetFourVelocity(vpcon, geom, CellLocation::Cent, k, j, i, ucon);
+
+        // Radiation
+        if (do_rad) {
```

Comment:
This is not necessarily what we're always going to want. Should it specify equipartition or something?

Reply:
Yeah the description of this PR even says that I let the user choose between thermal initial radiation and no initial radiation lol

Reply:
OK no initial radiation should work as well now